gateway/gateway-runtime/policy-engine/internal/analytics/publishers/log_test.go

```
@@ -628,18 +651,20 @@ func TestLog_Publish_GlobalFallback_NoPropertiesConfiguredOmitsKey(t *testing.T)
 		Enabled:    true,
+		Outputs:    []string{config.TrafficLogSinkStdout},
 		Properties: map[string]string{"apiName": "$ctx:api.name"},
 	})
+	require.NoError(t, err)
 	require.Nil(t, l.globalDir.Properties, "globalDir must never carry baked-in properties")
 
 	readOnce := func(event *dto.Event) map[string]interface{} {
 		path := filepath.Join(t.TempDir(), "out.log")
 		f, err := os.Create(path)
 		require.NoError(t, err)
 		defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)